MC/MC.py

```diff
@@ -48,7 +48,7 @@ def import_psyphysdata_mc(f_name, path):
         coh.append(float(trl_dat[2]))
         isOK.append(int(trl_dat[3]))
         if trl_dat[4] == 'space' or trl_dat[4] == '-':
-            key.append(0)
+            key.append(-2)
         else:
             key.append(int(trl_dat[4]))
```

Comment:
thiis quite cryptic , e.g. why 0 was incorrect and `-2` is so much better etc....  leave comments for yourself so later on you could figure out why you did it this way.